### ISSUE TITLE
`MAILER_URL` can be invalid

### ISSUE THREAD

**Issue by thasmo** [platformsh/symfonyflex-bridge#20]
Symfony introduced a change lately which validates the `MAILER_URL` to be a valid URL (and throws an exception if not). This now fails on the `MAILER_URL` set by `symfonyflex-bridge` because it sets the `MAILER_URL` to an invalid URL if sending emails is disabled in the environment settings.

The problem occurs on [this line](https://github.com/platformsh/symfonyflex-bridge/blob/d1cb0b85d417e927a9acfa1a48aeb94945d458dd/platformsh-flex-env.php#L87), where `$config->smtpHost` is empty or not set, because email sending is disabled in the environment settings.

Looking at [this lines](https://github.com/platformsh/symfonyflex-bridge/blob/d1cb0b85d417e927a9acfa1a48aeb94945d458dd/platformsh-flex-env.php#L52-L54) it may makes sense to adapt them to this or something similar:
```php
if (!getenv('MAILER_URL') && $config->smtpHost) {
    mapPlatformShSwiftmailer($config);
}
```

**Comment by Aricura** [comment]
Additionally, if sending mails is disabled, the `MAILER_URL` should still be set, but using a `NULL` protocol instead of `SMTP` to ensure the `MAILER_URL` won't be set to the value defined in .env file.

**Comment by Crell** [comment]
Resolved by #21
